### PR TITLE
remove unecessary telemetry

### PR DESCRIPTION
**Pull request by ericawright**
Lots of telemetry has been removed, it's possible I missed something. 
remove measurement of blocked trackers
remove timing measurements
remove error measurements

fixes: #14

### CODE REVIEW

File: src/feature.js [hunk 1]
```diff
@@ -156,12 +150,6 @@ class Feature {
         this.recordSurveyInteraction(tabId, SURVEY_PAGE_NOT_BROKEN, disableStudyChecked);
       },
     );
-
-    browser.trackers.onErrorDetected.addListener(
-      (error, tabId) => {
-        this.recordPageError(error, tabId);
```
Comment (groovecoder) [inline src/feature.js]: We can also remove the `recordPageError` function completely.

File: test/functional/1-telemetry.js [hunk 2]
```diff
@@ -66,30 +56,6 @@ describe("telemetry", function() {
       assert.equal(attributes.user_reported_page_breakage, "false", "user reported page breakage exception is not included in the ping");
     });
 
-    it("correctly records performance metrics", async () => {
```
Comment (groovecoder) [inline test/functional/1-telemetry.js]: Heh, I was about to comment that this test was failing. 😜

File: docs/TELEMETRY.md [hunk 3]
```diff
@@ -41,26 +41,10 @@ telemetry: {
   "data": {
     "attributes": {
       "etld":"f231d141395abf6f4c98dd55fe8c37e2752e82d72e1ffd3b64bdc6c978692fc6",
-      "num_blockable_trackers":"2",
```
Comment (groovecoder) [inline docs/TELEMETRY.md]: According the PHD it seems like we still want this number? "# of trackers loaded (integer; could be 0?)" under https://docs.google.com/document/d/1JeCS71Kf0ZJivxgNrrIvpViHCwU-QOhKvH92Av0b7Yc/edit#heading=h.d47hkl2j4nbu

**Comment by ericawright** [comment]
thank you, updated

**Pull request merged**